lib/BabelPluginExportToFunction.js

```diff
@@ -8,18 +8,30 @@ const CALLBACK_ID = '__bundlerCallback';
 const CONFIG_ID = '__bundlerConfig';
 const EXPORT_ERROR = 'auth0-bundler could not resolve a function with at least 2 parameters as default export.';
 const buildArgumentIdentifier = (id) => `__bundlerArg${id + 1}`;
-const buildFunctionWrapper = babelTemplate(`function ${WRAPPER_ID}(FUNCTION_IDENTIFIERS) {
+const buildFunctionWrapperVar = babelTemplate(`function ${WRAPPER_ID}(FUNCTION_IDENTIFIERS) {
   'use strict';
 
   var ${CONFIG_ID} = CONFIG
   BODY
 }`);
-const buildModuleWrapper = babelTemplate(`module.exports = function ${WRAPPER_ID}(FUNCTION_IDENTIFIERS) {
+const buildModuleWrapperVar = babelTemplate(`module.exports = function ${WRAPPER_ID}(FUNCTION_IDENTIFIERS) {
     'use strict';
 
     var ${CONFIG_ID} = CONFIG
     BODY
 };`);
+const buildFunctionWrapperConst = babelTemplate(`function ${WRAPPER_ID}(FUNCTION_IDENTIFIERS) {
+  'use strict';
```

Comment:
💅 I think the indentation should be 4 spaces until line 27.

Reply:
fixed